server/athenian/api/controllers/team_controller.py

```diff
@@ -2,15 +2,18 @@
 from itertools import chain
 import logging
 from sqlite3 import IntegrityError, OperationalError
-from typing import Any, Dict, Iterable, List, Mapping, Optional, Tuple, Union
+from typing import Any, Dict, Iterable, List, Mapping, Optional, Sequence, Tuple, Union
 
 from aiohttp import web
 import aiomcache
 from asyncpg import UniqueViolationError
 import morcilla
+import sqlalchemy as sa
 from sqlalchemy import and_, delete, insert, select, update
+from sqlalchemy.orm.attributes import InstrumentedAttribute
 
 from athenian.api import metadata
+from athenian.api.align.exceptions import TeamNotFoundError
```

Comment:
Let's move this exception to the team controller for now. We should properly refactor later ofc, but we will at least avoid importing align inside the controllers and the potential circular imports.